### ISSUE TITLE
x86 version?

### ISSUE THREAD

**Issue by Nicolas-Ar** [CMuncey/OoT_Compressor#5]
Not an issue, is a request only if possible.
can you compile the project to be compatible with 32 bit architecture?

Decompressor works flawless with Majora's Mask!!!
Thank you for this tool.

**Comment by CMuncey** [comment]
Made a new release. That one should work, let me know if it doesn't.

**Comment by Nicolas-Ar** [comment]
> 
> 
> Made a new release. That one should work, let me know if it doesn't.

Sorry for the late response, ive tried on USA and Europe of Majora's Mask ROM with some changes and works like a charm under win 7 x86 (both tools), i have to mention one problem, but i think is more related with the program i use to translate, at some point corrupts the expanded rom because after compressing it in the intro when the mask is floating gives me a black screen only with sound....

Anyways once again many thankies for this!

**Comment by CMuncey** [comment]
That's an issue with emulators trying to emulate the "blur" effect they added in MM I believe. Try turning on frame buffer if it's off, that should help according to Krim

Another note, after re-reading your post. If the CRC in the rom isn't identical and you're using pj64, that can also cause it to do that. pj64 has some dumb hacks to make it's games work properly, and it applies those hacks based on the CRC of the ROM. I'm not entirely sure how to fix that one other than switching emulators. Maybe manually setting the CRC to the proper value in the ROM with a hex editor would work too

**Comment by Nicolas-Ar** [comment]
Yeah i use pj64 only for testing and view quick changes (fast cheats, graphics) rather than playing on it.
So, i will follow your advices and report it back more testing but at moment there's no any issue related about your tool.

Sorry if i'm not explaining well or saying the things more clear.
